src/status_im/ui/screens/communities/community_overview_redesign.cljs

```diff
@@ -48,121 +50,217 @@
                                  {:name "announcements"
                                   :emoji "🎺"}]
                        :Mobile [{:name "mobile"
-                                 :emoji "👽"}]})
-
-(defn channel-list-component []
-  [rn/scroll-view {:style {:margin-top 20}}
-   [:<> {:style {:flex 1}}
-    (map (fn [category]
-           ^{:key (get category 0)}
-           [rn/view {:flex 1}
-            [divider-label/divider-label
-             {:label (first category)
-              :chevron-position :left}]
-            [rn/view
-             {:margin-left   8
-              :margin-top    10
-              :margin-bottom 8}
-             [list/flat-list
-              {:shows-horizontal-scroll-indicator false
-               :separator                         [rn/view {:margin-top 4}]
-               :data                              ((first category) list-of-channels)
-               :render-fn                         channel/list-item}]]])
-         list-of-channels)]])
+                                 :emoji "👽"}
+                                {:name "mobile-ui"
+                                 :emoji "👽"}
+                                {:name "mobile-ui-reviews"
+                                 :emoji "👽"}]
+                       :Desktop [{:name "desktop"
+                                  :emoji "👽"}
+                                 {:name "desktop-ui"
+                                  :emoji "👽"}
+                                 {:name "desktop-ui-reviews"
+                                  :emoji "👽"}
+                                 {:name "desktop2"
+                                  :emoji "👽"}
+                                 {:name "desktop-ui2"
+                                  :emoji "👽"}
+                                 {:name "desktop-ui2-reviews"
+                                  :emoji "👽"}]})
+
+(defn channel-list-component [channel-heights first-channel-height]
+  [rn/view {:on-layout #(swap! first-channel-height
+                               (fn [] (int (Math/ceil (-> % .-nativeEvent.layout.y)))))
+            :style {:margin-top 20 :flex 1}}
+   (map (fn [category]
+          ^{:key (first category)}
```

Comment:
We can use a `let` to save the first category here since we use in multiple places ..

Reply:
sounds good